CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
 # Change Log
 
 ## [Unreleased]
-- Added "observer" attribute to @property annotation
-- Added "computed" attribute to @property annotation
-- Fix bug where use of `Polymer.mixinBehaviors` with `@customElement` would throw an exception relating to the definition of the element's "properties" property.
+
+## [0.1.2] - 2018-01-01
+- Added `observer` and `computed` attribute to `@property` decorator.
+- Added `@listen` decorator and `DeclarativeEventListeners` mixin.
+- Fixed bug where use of `Polymer.mixinBehaviors` with `@customElement` would throw an exception relating to the definition of the element's "properties" property.
```

Comment:
I think it was with the `@property` decorator, no?

Reply:
Good point. Or `@computed`. Updated.